package.json

```diff
@@ -36,6 +36,7 @@
   "devDependencies": {
     "browserify": "^10.2.4",
     "chai": "~3.0.0",
+    "flat": "^2.0.1",
```

Comment:
This should be in `dependencies`. Otherwise npm will not download it as deep dependency.